src/uu/expand/src/expand.rs

```diff
@@ -89,23 +89,36 @@ fn tabstops_parse(s: &str) -> (RemainingMode, Vec<usize>) {
                 }
                 _ => {
                     // Parse a number from the byte sequence.
-                    let num = from_utf8(&bytes[i..]).unwrap().parse::<usize>().unwrap();
+                    let s = from_utf8(&bytes[i..]).unwrap();
+                    if let Ok(num) = s.parse::<usize>() {
+                        // Tab size must be positive.
+                        if num == 0 {
+                            crash!(1, "tab size cannot be 0");
```

Comment:
can you add a test when it is 0?
also, is crash the best option to exit here?

Reply:
Yes, I can add such a test.

It would probably be cleaner to make the function return a result.

Reply:
Ok, in the latest push I replaced the `crash!` statements and return a result instead.